src/Robo/Commands/Tests/PhpUnitCommand.php

```diff
@@ -2,29 +2,13 @@
 
 namespace Acquia\Blt\Robo\Commands\Tests;
 
-use Acquia\Blt\Robo\BltTasks;
 use Acquia\Blt\Robo\Exceptions\BltException;
-use Robo\Contract\VerbosityThresholdInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 /**
  * Defines commands in the "tests" namespace.
  */
-class PhpUnitCommand extends BltTasks {
-
-  /**
-   * Directory in which test logs and reports are generated.
-   *
-   * @var string
-   */
-  protected $reportsDir;
-
-  /**
-   * The filename for PHPUnit report.
-   *
-   * @var string
-   */
-  protected $reportFile;
+class PhpUnitCommand extends DrupalTestCommand {
```

Comment:
The class inheritance here feels a little funny to me. I think I understand the _why_, but it almost feels like a `Car` is extending an `Accord` if that makes sense...?